ecl/hql/hqlgram2.cpp

```diff
@@ -11763,6 +11763,7 @@ static void getTokenText(StringBuffer & msg, int token)
     case TOK_TRUE: msg.append("TRUE"); break;
     case TYPE: msg.append("TYPE"); break;
     case TYPEOF: msg.append("TYPEOF"); break;
+    case UNCOMPRESSED: msg.append("UNCOMPRESSED"); break;
```

Comment:
Also needs to be added to reservedwords.cpp

Reply:
will add